.github/workflows/test.yaml

```diff
@@ -21,13 +21,14 @@ jobs:
       - uses: actions/setup-go@v3
         with:
           go-version: "1.19"
+          cache: true
```

Comment:
How about we upgrade to v4 of the plugin instead? It appears to enable cache by default

https://github.com/actions/setup-go#v4

Reply:
Oh wow, v4 came out 2 hours ago. That's convenient! Yes agreed

Reply:
Turns out they added it in the readme, but haven't actually cut a release and published it. Couldn't resolve v4